api/src/main/java/io/grpc/ClientCall.java

```diff
@@ -279,4 +279,14 @@ public void setMessageCompression(boolean enabled) {
   public Attributes getAttributes() {
     return Attributes.EMPTY;
   }
+
+  /**
+   * Returns tracer specific properties (if any) attached by tracing components.
```

Comment:
Add a link to `ClientCallTracer`.
Also say that this is available as soon as the ClientCall is created.